robottelo/hosts.py

```diff
@@ -430,20 +519,252 @@ def set_infrastructure_type(self, infrastructure_type='physical'):
         script_content = '\n'.join(script_content)
         self.execute(f"echo -e '{script_content}' > {script_path}")
 
-    def patch_os_release_version(self, distro=DISTRO_RHEL7):
+    def patch_os_release_version(self, distro=constants.DISTRO_RHEL7):
         """Patch VM OS release version.
 
         This is needed by yum package manager to generate the right RH
         repositories urls.
         """
-        if distro == DISTRO_RHEL7:
-            rh_product_os_releasever = REPOS['rhel7']['releasever']
+        if distro == constants.DISTRO_RHEL7:
+            rh_product_os_releasever = constants.REPOS['rhel7']['releasever']
         else:
             raise ContentHostError('No distro package available to retrieve release version')
         return self.execute(f"echo '{rh_product_os_releasever}' > /etc/yum/vars/releasever")
 
+    # What the heck to call this function?
+    def host_populate(
```

Comment:
maybe this to be consistent with capsule_setup?
```suggestion
    def contenthost_setup(
```